mechanical/rocker_bogie/README.md

```diff
@@ -111,9 +111,9 @@ Attach another 3.75 inch channel **S3** to the bottom hole of the pattern F brac
 
 ### 4.5 Opposite edge of Rocker-Bogie Joint
 
-Attach another Pattern F bracket **S7** to the opposite side of the channels using washers **W1** and **W3**, screws **B4**, and hex nuts **B8** as shown in Figure 7.
+Attach another Pattern F bracket **S7** to the opposite side of the channels using washers **W1** and **W3**, screws **B2**, and hex nuts **B8** as shown in Figure 7.
 
-| <img src="../../images/rocker_bogie/step_8.png" height="300"> <img src="../../images/rocker_bogie/step_9b.png" height="300"> |
+| <img src="../../images/rocker_bogie/step_1.png" height="300"> <img src="../../images/rocker_bogie/step_9b.png" height="300"> |
```

Comment:
`step_1.png` is already used in section 4.1. I took a look at the [documentation from before we switched to Markdown](https://github.com/nasa-jpl/open-source-rover/blob/ff002ecdcb855e3f69c6eadd58648b2c32064566/mechanical/rocker_bogie/Rocker-Bogie%20Build%20Doc.pdf) and there's only one image there.

So the options are to delete this image and just have one image instead, or if you think it's helpful to have some image there, let's find/create an image that would be good.

Reply:
I was thinking it would be useful to show the mirror image.  I did just add a physical image in and update the reference.  I think it is pretty clear to basically just mirror the other side.  See what you think...it is not a CAD model pic.